backend/czi_hosted/common/rest.py

```diff
@@ -260,7 +260,9 @@ def diffexp_obs_post(request, data_adaptor):
     try:
         # TODO: implement varfilter mode
         mode = DiffExpMode(args["mode"])
-
+        two_lists = False
```

Comment:
why do we need a mode switch?  Why not just remove the old code, and support only the "topN/bottomN" implementation?

this would simplify tests & code.